### ISSUE TITLE
nbconvert from jupyter notebook to pdf cell color not detected

### ISSUE THREAD

**Issue by arun-siv** [jupyter/nbconvert#623]
There is a clear cell color for [in] and [out]. Those are not detected when converted to pdf

**Comment by arun-siv** [comment]
[How To Build DataFrame.pdf](https://github.com/jupyter/nbconvert/files/1133787/How.To.Build.DataFrame.pdf)
I am uploading a file as example. It doesnt show the cell color for the [in]

**Comment by takluyver** [comment]
I'm not quite sure what you mean - do you want the grey background for input code?

**Comment by arun-siv** [comment]
@takluyver  Yes,  I wanted a grey background for input code . Also the table borders are not visible. It converts it as dash lines

**Comment by arun-siv** [comment]
Even the table borders not appearing in the pdf output . There is not wrapping of the cell too .

**Comment by takluyver** [comment]
I think not wrapping the code cells is a bug. The rest is just style questions - it goes via latex, so it doesn't look like the HTML option.

**Comment by parthi2929** [comment]
this issue was not solved yet?

**Comment by t-makaro** [comment]
Wrapping code cells is not really possible without greatly refactoring the latex implementation. But, I have done this and fixed the style issues over at https://github.com/t-makaro/nb_pdf_template.

**Comment by parthi2929** [comment]
I was trying to get the code cell part with boxed grey background.. there is no style_jupyter in my anaconda nbconvert.. i have been trying for past few hours but some issue as am new to latex.. no error but am also not getting the code cell boxed..  is it possible to give template for that alone.. my current template is [here](https://gist.github.com/parthi2929/809f99a616b97ffe947df148cf525989)

**Comment by parthi2929** [comment]
you have created new environments like ```InVerbatim``` that seem to work individually ( a simple MWE [here](https://gist.github.com/parthi2929/2855a9c1792c7ff0ebe67205ee023999)) when I call but nbconvert does not seem to deploy it anyway in final converted code.. what is missing?

**Comment by t-makaro** [comment]
You could just inherit from ```style_jupyter```, and overwrite some of my other modifications. I do recommend using the ```OutVerbatim``` as well (it won't color the background, but it is need to format consistently). You would get a template like:

```
((* if not cell_style is defined *))
    ((*- set cell_style = 'style_jupyter.tplx' -*))
((* endif *))
((=- Inherit from the specified cell style. -=))
((* extends cell_style *))

((* block repygments *))((* end block repygments *))

((* block docclass *))
\documentclass[11pt]{article}
((*- endblock docclass -*))
```
All of my other modifications are in classic.tplx or classicm.tplx (except the syntax highlighting changes). You may want to modify the prompt as well with a ```\noindent``` since you won't be using ```\usepackage{parskip}```:
```
((*- block style_prompt *))
    \newcommand{\prompt}[3]{
        \needspace{1.1cm}
        \settowidth{\promptlength}{ #1 [#3] }
        \ifleftmargins\hspace{-\promptlength}\hspace{-5pt}\fi
        \noindent{\color{#2}#1 [#3]:}
        \ifleftmargins\vspace{-2.7ex}\fi
    }
((* endblock style_prompt *))
```

**Comment by parthi2929** [comment]
I tried that, Since I have other customizations I had to tinker your ```style_jupyter.tplx``` more which was cumbersome (mostly because I am new to these stuff), so I am trying to keep my existing customizations as they are, and adding your box part alone for now. I almost get it, but the tex file ends up having "In Verbatim" instead of "InVerbatim", the space comes from value you are supplying as "In " while calling draw_cell I guess, do not know how to fix that (that space might be there for a reason? when I tried using your ```style_jupyter.tex``` this problem did not occur but conflicts with my other customizations)

**Comment by t-makaro** [comment]
**CC** @MSeal. I think this should be closed by #992 as well.
